values-pt-rBR/strings.xml

```diff
@@ -1,78 +1,78 @@
 <resources tools:ignore="ExtraTranslation" xmlns:tools="http://schemas.android.com/tools">
-    <string name="translators">"@abacate123"</string> <!--put/add yourself here as the AUTHOR(S) of this translation-->
+    <string name="translators">"@caiostob"</string> <!--put/add yourself here as the AUTHOR(S) of this translation-->
```

Comment:
Thanks for updating the whole file, I still feel it would be respectful and appropriate to also keep the name of @abacate123 here, since he did certainly do the initial work on the 1.0 translation.